src/github/client/github-installation-client.ts

```diff
@@ -240,10 +242,29 @@ export class GitHubInstallationClient extends GitHubClient {
 	};
 
 	public listDeployments = async (owner: string, repo: string, environment: string, per_page: number): Promise<AxiosResponse<Octokit.ReposListDeploymentsResponse>> => {
-		return await this.get<Octokit.ReposListDeploymentsResponse>(`/repos/{owner}/{repo}/deployments`,
-			{ environment, per_page },
-			{ owner, repo }
-		);
+		try {
+			return await this.get<Octokit.ReposListDeploymentsResponse>(`/repos/{owner}/{repo}/deployments`,
+				{ environment, per_page },
+				{ owner, repo }
+			);
+		} catch (e) {
+			try {
+				if (await booleanFlag(BooleanFlags.LOG_CURLV_OUTPUT, this.jiraHost)) {
+					this.logger.warn("Found error listing deployments, run curl commands to get more details");
+					const { headers } = await this.installationAuthenticationHeaders();
+					const { Authorization } = headers as { Authorization: string };
+					const output = await runCurl({
+						fullUrl: `${this.baseUrl}/repos/{owner}/${repo}/deployments`,
+						method: "GET",
+						authorization: Authorization
+					});
+					this.logger.warn({ body: output.body, meta: output.meta }, "Curl for list deployments output generated");
```

Comment:
I think we should remove authorization header from output, that's too dangerous... wdyt?

Reply:
nevermind, see you already doing that